src/api/v2/notifications/types.ts

```
@@ -30,10 +33,18 @@ export type InviteJoinRequestNotificationData = {
   autoApprove: boolean;
 };
 
+export type CreditsRefilledNotificationData = {
+  creditsAdded: number;
+  newBalance: string;       // bigint serialized
```

ESLint

[error] Delete `······`

(prettier/prettier)